pyproject.toml

```diff
@@ -54,6 +54,13 @@ dev = [
     "ruff==0.0.290",
 ]
 
+makani = [
+    "makani @ git+https://github.com/NVIDIA/modulus-makani.git@v0.1.0",
+    "torch_harmonics @ git+https://github.com/NVIDIA/torch-harmonics.git@v0.6.5",
```

Comment:
Oh! I am not sure if this is going to break our pypi uploads... In the past we have had issues with such git links. Refer: https://github.com/pypa/twine/issues/726 https://github.com/NVIDIA/modulus/pull/64

Reply:
Is there a possibility of the torch_harmonics and makani team releasing pypi wheels?

Reply:
That's a very good point! I saw the [Attention](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#direct-url-dependencies) notice in the docs but mistakenly thought it's only related to mandatory dependencies, not optional.
Let me discuss it with Makani folks.
For now, here is what we can do:
1. Comment out (with TODO comment) these direct URL dependencies.
2. Install them in Dockerfile CI stage (so the tests could run)

Once Makani is in pypi we can update the code accordingly.

Thought?

Reply:
torch-harmonics can be installed through pypi already. Are we sure that this is breaking?

makani requires some workarounds at the moment to fix some issues with complex numbers in PyTorch. This requires overwriting some files in the torch installation and as long as this is the case, I'd prefer having it as an "experimental" repository that needs to be installed through git.

Reply:
Yes, I am reasonably confident that having the git links in the optional dependency will also be flagged. The solution suggested by @Alexey-Kamenev sounds like a good alternative (installing it via git inside the Dockerfile). Additionally, we can also add appropriate docs to the READMEs of Makani/Modulus (wherever appropriate) to mention this step to the users.

Reply:
That sounds like a reasonable workaround.